lib/bas/bot/fetch_birthdays_from_notion.rb

```diff
@@ -81,13 +88,22 @@ def body
 
       {
         filter: {
-          or: [
-            { property: "BD_this_year", date: { equals: today } }
-          ]
+          and: [{ property: "BD_this_year", date: { equals: today } }] + last_edited_condition
         }
       }
     end
 
+    def last_edited_condition
```

Comment:
Could this be a common utility for fetching bots? 🤔

Reply:
Hello @L-Zuluaga, Yes, it could be a common utility for fetching bots from Notion, but it depends on the use case and the limitations of the Notion API. The Notion API has restrictions on how many "and/or" operations can be included in a single query (2). If this limitation is exceeded, the API returns an error response. That’s why I added the functions in some bots.